ta4j-core/src/main/java/org/ta4j/core/num/DoubleNum.java

```diff
@@ -36,8 +38,9 @@ this software and associated documentation files (the "Software"), to deal in
 
     private final double delegate;
 
-    private final static double EPS = 0.00001; // precision
-
+    //private static final double EPS = 0.000000000000001;
+    private static final double EPS = TestUtils.GENERAL_OFFSET;
```

Comment:
I would prefer to swith dependency ( TestUtils.GENERAL_OFFSET = DoubleNum.EPS). The main package (core) should be able to run without test package

Reply:
Okay.